docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,196 +1,222 @@
+Documentation SHOULD aim for a lower secondary education reading level, as recommended by the [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=315#readable). |  |
+Providing a translation of any code, documentation or tests is OPTIONAL. |  |
 
 ## [Use open standards](https://standard.publiccode.net/criteria/open-standards.html)
 
-- [ ] compliant with this criterion.
+- [x] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
-For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the Open Source Initiative Open Standard Requirements. |  | [StUF-ZKN](https://www.gemmaonline.nl/index.php/Sectormodel_Zaken:_StUF-ZKN) OpenAPI2 spec, GraphQL, Oauth2, OpenID Connect, SMTP, JSON?
-If no existing open standard is available, effort SHOULD be put into developing one. |  | Maybe we need a standard way to forward the complaint between bodies, e.g.: City <--> Province?
+For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr). | yes | [standards](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md#standards)
+Any non-open standards used MUST be recorded clearly as such in the documentation. | N/A |
+Any standard chosen for use within the codebase MUST be listed in the documentation with a link to where it is available. | yes | [standards](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md#standards)
+Any non-open standards chosen for use within the codebase MUST NOT hinder collaboration and reuse. | N/A |
+If no existing open standard is available, effort SHOULD be put into developing one. | N/A |
 Standards that are machine testable SHOULD be preferred over those that are not. | N/A |
-Functionality using features from a non-open standard (one that doesn’t meet the Open Source  Initiative Open Standard Requirements) MAY be provided if necessary, but only in addition to compliant features. |  |
-All non-compliant standards used MUST be recorded clearly in the documentation. | N/A |
-The codebase SHOULD contain a list of all the standards used with links to where they are available. | no | Created [backend#65](https://github.com/Signalen/backend/issues/65).
 
 ## [Use continuous integration](https://standard.publiccode.net/criteria/continuous-integration.html)
 
-- [ ] compliant with this criterion.
+- [ ] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
-All functionality in the source code MUST have automated tests. | mostly | In addition to unit testing, there are end-to-end tests which run on master and release branches (frontend); there are coverage reports, results require manual download.
+All functionality in the source code MUST have automated tests. |  | Would like to see code coverage published
```

Comment:
Code coverage is generated by the test runner, available to people with enough access to Github. Note that other users can run the exact same test suite and derive the same numbers (as test suite is published).

Reply:
Indeed, coverage can be seen by diving into the checks, e.g.:
https://github.com/Amsterdam/signals/pull/1071/checks